Code/Source/RobotImporter/RobotImporterWidget.cpp

```diff
@@ -6,18 +6,63 @@
  *
  */
 
+#include <QMessageBox>
 #include <QPushButton>
 #include <QVBoxLayout>
 
 #include "RobotImporter/RobotImporterWidget.h"
-#include "RobotImporter/URDF/URDFPrefabMaker.h"
+
+#include <AzCore/Utils/Utils.h>
 
 namespace ROS2
 {
+    namespace Internal
+    {
+        AZStd::optional<QString> GetPathWithExtension(
```

Comment:
The name could be changed to something that better matches the behavior 
(interaction with a user through file dialog)